ingestion/src/metadata/clients/powerbi_client.py

```diff
@@ -91,18 +92,30 @@ def fetch_dashboards(self) -> Optional[dict]:
 
         return None
 
-    def fetch_all_workspaces(self) -> Optional[dict]:
+    def fetch_all_workspaces(self) -> List[dict]:
```

Comment:
The type Hint should be Optional[List[dict]]

Reply:
Yes thanks

Reply:
@TeddyCr added the `Optional`